### PR TITLE
[dev-v2.7] RKE2 patch releases

### PR DESCRIPTION
**Pull request by brooksn**
https://github.com/rancher/rancher/issues/43231

### CODE REVIEW

**Review by rancher-max — COMMENTED** [review]
Make sure to remove the September patches as part of this PR also please!

**Review by rancher-max — COMMENTED** [review]
~Wait, nevermind on my previous comment. That is only relevant for 2.8 I think~

File: channels-rke2.yaml [hunk 1]
```diff
@@ -1496,14 +1527,45 @@ releases:
...
+        type: enum
+        options:
+        - cis
+        - cis-1.23
+        nullable: true
+    charts: &charts-v1-26-10-rke2r1
       <<: *charts-v1-26-8-rke2r1
       rke2-cilium:
         repo: rancher-rke2-charts
         version: 1.14.100
```
Comment (rancher-max) [inline channels-rke2.yaml]: Let's remove this since we have it defined later on

File: channels-rke2.yaml [hunk 2]
```diff
@@ -1390,16 +1390,47 @@ releases:
...
+        type: enum
+        options:
+        - cis
+        - cis-1.23
+        nullable: true
+    charts: &charts-v1-25-15-rke2r1
       <<: *charts-v1-25-13-rke2r1
       rke2-cilium:
         repo: rancher-rke2-charts
         version: 1.14.100
```
Comment (rancher-max) [inline channels-rke2.yaml]: Let's remove this since we have it defined later on

**Review by rancher-max — APPROVED** [review]

**Review by briandowns — APPROVED** [review]

**Pull request merged**